docs/topics/api/overview.rst

```diff
@@ -0,0 +1,88 @@
+.. _api-overview:
+
+========
+Overview
+========
+
+This describes the details of the requests and responses you can expect from
+the `addons.mozilla.org <https://addons.mozilla.org/en-US/firefox/>`_ API.
+
+--------
+Requests
+--------
+
+All requests should be made with the header::
+
+        Content-type: application/json
+
+---------
+Responses
+---------
+
+~~~~~~~~~~~~
+Status Codes
+~~~~~~~~~~~~
+
+There are some common API responses that you can expect to receive at times.
+
+.. http:get:: /api/v3/...
+
+    :statuscode 401: Authentication is required or failed.
+    :statuscode 403: You are not permitted to perform this action.
+    :statuscode 404: The requested resource could not be found.
+    :statuscode 500: An unknown error occurred.
+    :statuscode 503:
+        The site is in maintenance mode and this operation is not permitted.
+
+
+~~~~~~~~~~
```

Comment:
This heading style is a bit ugly imho (the bit on top of the heading is optional) - but I guess it's already in-use so to be consistent it should stay.